Fixing typo and formatting (#41621)
keycloak/keycloak: docs/guides/securing-apps/token-exchange.adoc

Token exchange request sent with `scope=optional-scope2` and with `audience=target-client2`

Same like previous example, but `target-client1` audience and client roles filtered due audience parameter was included, but only with this `target-client2` client. The client scope `default-scope1` will be

> also filtered due it contains some client roles, but at the same time, it does not contain any client roles of requested audience client `target-client2`. So the token would be like:

Reviewer: ```suggestion
also filtered because it contains some client roles but no client roles of the requested audience client `target-client2`. So the token would be as follows:
```

s|Internal-internal token exchange | Supported. Implemented as per rfc8693 | Preview support. Loose implementation of rfc8693. It is recommended to use V2 instead
s|Allowed `subject_token_type`     | Access token type only   | Access token type only for internal-internal, JWT for external-internal scenarios
s|Allowed `requested_token_type`   | Access token (default), Refresh token, ID token | Access token, Refresh token (default), SAML2 assertion

> s|Behaviour of `scope` parameter   | Aligned with other grants. Scope parameter means requesting optional scopes of the client, which sent the token exchange request  | Scope parameter based on the scopes of

Author: Yes, I'll fix it to use the US one.
